src/sep6/DepositProvider.ts

```diff
@@ -0,0 +1,65 @@
+import queryString from "query-string";
+
+import { Omit } from "../util";
+import { fetchKycInBrowser } from "./fetchKycInBrowser";
+import { TransferProvider } from "./TransferProvider";
+import {
+  DepositRequest,
+  FeeArgs,
+  InteractiveKycNeeded,
+  TransferResponse,
+} from "./types";
```

Comment:
In my last revision, I had the types inherit from `src/types/index.ts` which imports `src/types/data.ts` and `src/types/keys.ts`. Maybe those should be .d.ts files instead, but either way, we should have all three arms be consistent with each other.